src/_js/map.js

```diff
@@ -181,6 +181,43 @@
 			}
 		});
 	};
+	TibiaMap.prototype._loadMarkers = function () {
+		const _this = this;
+		const icons = []
+		const symbols = ["!", "$", "?", "bag", "checkmark", "cross", "crossmark", "down", "flag", "lock", "mouth", "red down", "red left", "red right", "red up", "skull", "spear", "star", "sword", "up"];
+		symbols.forEach(s => {
+			icons[s] = L.icon({iconSize: [11, 11], iconUrl: s.replace("!", "exclamation").replace("$", "dollar").replace("?", "question").replace(" ", "_") + ".png"});
```

Comment:
Nit: the other resources on TibiaMaps.io use `-` instead of `_` in file names. Would it be okay to rename the image files accordingly? And then just change the `replace` here:

```suggestion
			icons[s] = L.icon({iconSize: [11, 11], iconUrl: s.replace('!', 'exclamation').replace('$', 'dollar').replace('?', 'question').replace(' ', '-') + '.png'});
```

Reply:
I'm wondering if we'll need to apply CSS `image-rendering: pixelated` on these images for high-DPI screens.

Reply:
Applying it actually made them better on my screen :)

Reply:
Nice!